libs/ui/packages/react/src/pre-ldls/libs/index.ts

```diff
@@ -5,6 +5,16 @@ import { dark, light, ModeColors, spacing, SpacingScale } from "./design-tokens"
 // Add temporarily missing tokens here
 const extraOther = {
   "radius-s": "8px",
+  "radius-xs": "4px",
+  "margin-xs": "8px", // redefines marging-xs
+  "margin-s": "16px", // redefines marging-s
+  "margin-m": "24px", // redefines marging-m
+  "margin-l": "32px", // redefines marging-l
+} as const;
+
+const overrideOther = {
+  "spacing-xs": "12px", // from "spacing-xxs": "12px"
+  "spacing-xxs": "8px", // from "spacing-xs": "8px"
 } as const;
```

Comment:
Its a small detail but IMO one variable "overrideOther" with all the keys is enough no need to split them in 2 objects.
Thanks for clarifying the changes with the comments 👍.
Also we should check on whether the ldls design tokens have been updated (or at least fixed).

Reply:
merged extraOther and overrideOther. Confirmed [design-tokens.ts](https://github.com/LedgerHQ/ldls/blob/main/libs/design-tokens/src/lib/design-tokens.ts) hasn't been updated.